go.mod

```diff
@@ -5,6 +5,8 @@ go 1.20
 require (
 	github.com/golang/mock v1.6.0
 	github.com/mattn/go-isatty v0.0.19
+	github.com/muesli/mango-cobra v1.2.0
```

Comment:
Do we need to update https://github.com/openfga/cli/blob/main/.golangci.yaml?

Reply:
It has been updated in this PR